tests/fs-test.rs

```diff
@@ -38,25 +38,52 @@ impl AsStr for &str {
 	}
 }
 
-fn create_uhyvefilemap_params<T: AsStr, U: AsStr>(host_path: T, guest_path: U) -> Vec<String> {
+// Creates a vector out of a given host path and guest path for UhyveFileMap.
+fn create_filemap_params<T: AsStr, U: AsStr>(host_path: T, guest_path: U) -> Vec<String> {
 	[format!("{}:{}", host_path.as_str(), guest_path.as_str())].to_vec()
 }
 
-/// This checks whether a VM can create and write to a file on the host.
+fn create_envvars<T: AsStr, U: AsStr>(testname: T, filepath: U) -> EnvVars {
+	EnvVars::Set(HashMap::from([
+		("TESTNAME".to_owned(), testname.as_str().to_owned()),
+		("FILEPATH".to_owned(), filepath.as_str().to_owned()),
+	]))
+}
+
+/// Generates a filename in the format of prefixab1cD23.txt
+fn generate_filename(prefix: &str) -> String {
+	let mut filename = prefix.to_owned();
+	let randomchar: String = rand::rng()
+		.sample_iter(&Alphanumeric)
+		.take(7)
+		.map(char::from)
+		.collect();
+	filename.push_str(&randomchar);
+	filename.push_str(".txt");
+	filename
+}
+
+/// Checks whether guests can create, then use files on the host.
+/// (The file is present in a mapped parent directory.)
 #[test]
-#[serial]
 fn create_mapped_parent_nonpresent_file() {
 	env_logger::try_init().ok();
+
+	let testname: &'static str = "create_mapped_parent_nonpresent_file";
+	let filename = generate_filename(testname);
+
 	// Tests successful directory traversal starting from file in child
 	// directory of a mapped directory.
-	let guest_path = PathBuf::from("/root/");
-	let mut host_path = get_fs_fixture_path();
-	host_path.push("ignore_everything_here");
-	let mut file_path = host_path.clone();
-	file_path.push("foo.txt");
-	remove_file_if_exists(&file_path);
+	let guest_dir_path: PathBuf = PathBuf::from("/root/");
+	let mut host_dir_path = get_fs_fixture_path();
+	host_dir_path.push("ignore_everything_here");
+
+	let mut host_file_path = host_dir_path.clone();
+	host_file_path.push(&filename);
+	let mut guest_file_path = guest_dir_path.clone();
+	guest_file_path.push(&filename);
```

Comment:
Looks like this can be moved in a separate fn to reduce code duplication.

Reply:
Not always, because we don't always use `host_file_path` and `guest_file_path` in `Params`'s `file_mapping`. I could create two separate functions that return whichever vector is necessary?

Reply:
It is not important, but it looks to me as if the same logic is at least used in several tests. So, maybe it can be pulled out only for those?